src/common/lib/client/realtimechannel.ts

```diff
@@ -83,6 +87,10 @@ class RealtimeChannel extends Channel {
     this.connectionManager = realtime.connection.connectionManager;
     this.state = 'initialized';
     this.subscriptions = new EventEmitter();
+    this.filteredSubscriptions = new Map<
```

Comment:
This will affect platform support, we should initialise the map only when `_subscribeFilter` is called for the first time